### PR TITLE
Single nuget package

### PR DESCRIPTION
**Pull request by kirankumarkolli**
Single Cosmos Nuget package (Includes Direct & HR).

### CODE REVIEW

File: Microsoft.Azure.Cosmos/src/Microsoft.Azure.Cosmos.csproj [hunk 1]
```diff
@@ -25,7 +28,7 @@
     <PackageIconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</PackageIconUrl>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos/src/Microsoft.Azure.Cosmos.csproj]: Undo

**Comment by kirankumarkolli** [comment]
Above are not ETA driven hence no point in adding them.

**Comment by ausfeldt** [comment]
> Open-sourcing above are not ETA driven.

Ok, then create issues without an ETA. After we open source Microsoft.Azure.Cosmos.Direct, then we can claim that the SDK is open-source. Until then it is half open-sourced.

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.NetFramework.Tests/GlobalEndpointManagerTest.cs [hunk 2]
```diff
@@ -59,21 +57,21 @@ public void EndpointFailureMockTest()
             GlobalEndpointManager globalEndpointManager = new GlobalEndpointManager(mockOwner.Object, connectionPolicy);
 
             globalEndpointManager.RefreshLocationAsync(databaseAccount).Wait();
-            Assert.IsTrue(globalEndpointManager.ReadEndpoints[0] == new Uri(readLocation1.Endpoint));
+            Assert.AreEqual(globalEndpointManager.ReadEndpoints[0], new Uri(readLocation1.Endpoint));
```
Comment (simplynaveen20) [inline Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.NetFramework.Tests/GlobalEndpointManagerTest.cs]: Although its minor  , new Uri(readLocation1.Endpoint) comes first as expected in assert?

Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.NetFramework.Tests/GlobalEndpointManagerTest.cs]: Expectation is first and current is consistent.

**Pull request merged**